### PR TITLE
Save given and family name separately on CFPs 

### LINKED ISSUE #997: Request first and last name separately when submitting talk proposals during cfp
Related to https://github.com/EuroPython/epcon/issues/988

### PR DESCRIPTION
**Pull request by rciorba**
Fixes #997
I've rebased the changes from #1072 and added a test for the name_tuple method

### CODE REVIEW

File: assopy/tests/test_assopy_user.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+from assopy.models import AssopyUser
+
+
+@pytest.mark.parametrize(
+    "first_name,last_name,expected", [
+        ("John", "Doe", ("John", "Doe")),
+        ("John Doe", "Doe", ("John", "Doe")),
+        # let's get some utf8 going
+        ("Miroslav", "Šedivý", ("Miroslav", "Šedivý")),
+        ("Miroslav Šedivý", "Šedivý", ("Miroslav", "Šedivý")),
```
Comment (umgelurgel) [inline assopy/tests/test_assopy_user.py]: Great way of test parametrization! I don't think we should have test cases based on an actual conference attendees though, worst case scenario, if he was to invoke GDPR, it's another thing to worry about.

**Review by umgelurgel — CHANGES_REQUESTED** [review]
Looks amazing! Left one tiny comment, otherwise should be ready to merge.

File: assopy/tests/test_assopy_user.py (hunk 1 above)
Comment (rciorba) [reply assopy/tests/test_assopy_user.py]: Fair enough :))

As I was thinking about unicode [Miro's talk](https://de.pycon.org/program/pydata-zna8wn-your-name-is-invalid-miroslav-sedivy/) came to mind  and I just couldn't help myself.

**Review by umgelurgel — APPROVED** [review]
Looks great! Thanks for applying the feedback :shipit:

**Pull request merged**